src/main/java/sum/MyCallable.java

```diff
@@ -0,0 +1,17 @@
+package sum;
+
+import java.util.List;
+import java.util.concurrent.Callable;
+
+public class MyCallable implements Callable<Integer> {
```

Comment:
Let's name class according to its' purpose

Reply:
Fixed